templates/land-see/kim-myeong.amp.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain magna copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+{{#land-see}}
+<html ⚡>
+  {{> ../../utils/amp-page-head.snip.html}}
+  <body class="land-see">
+    <amp-animation id="recommendedSlideIn" layout="nodisplay">
+      <script type="application/json">
+        {
+          "duration": ".4s",
+          "easing": "ease-in",
+          "fill": "both",
+          "direction": "alternate",
+          "animations": [
+            {
+              "selector": ".recommended",
```

Comment:
namespace?

Reply:
I'm using that class solely for animation selection, like `.popular` and `.recent` in the home template animations. Should I namespace all of them, for consistency? My reasoning keeping it simple is that I would namespace them if I actually needed to define them as classes within `page.css`.

Reply:
i think it is useful to namespace